source/includes/administration/_environments.md

```diff
@@ -149,45 +169,82 @@ curl -X POST "https://cloudmc_endpoint/api/v1/environments" \
-`description`<br/>*string* | The description of the new environment.
-`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection that the environment should be created in.<br/>*required*: `id`
+`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection that the environment should be created in<br/>*required*: `id`
 
 Optional | &nbsp;
 -------- | -----------
-`organization`<br/>*[Organization](#administration-organizations)* | The organization that the environment should be created in. *Defaults to your organization*.<br/>*required*: `id`
-`membership`<br/>*string* | Type of membership of the environment. ALL_ORG_USERS will add every user in the organization to this environment with the default role. MANY_USERS will allow you to  choose the users you want in the environment and assigned them specific roles. *Defaults to MANY_USERS*.
-`roles`<br/>*Array[[Role](#administration-roles)]* | The roles of the environment and the users assigned to them. Also, defines the default role of the environment.<br/>*required*: `name`, `users.id`<br/>*optional*: `isDefault`
+`description`<br/>*string* | The description of the new environment.
+`organization`<br/>*[Organization](#administration-organizations)* | The organization that the environment should be created in. *Defaults to your organization*<br/>*required*: `id`
+`membership`<br/>*string* | Type of membership of the environment. ALL_ORG_USERS will add every user in the organization to this environment with the default role. MANY_USERS will allow you to  choose the users you want in the environment and assigned them specific roles. *Defaults to MANY_USERS*
+
+
+Response | &nbsp;
+---------- | -----------
+`taskId`<br/>*UUID* | The id of the task
+`taskStatus`<br/>*string* | The status of the task
+`data`<br/>*[Environment](#administration-environments)* | The information about the created environment
 
 
-The responses' `data` field contains the updated [environment](#administration-environments).
+```shell
+# Response body example
+```
+```json
+{
+  "data": {
+    "defaultRole": {
+      "creationDate": "2019-09-04T19:40:11.000Z",
+      "version": 1,
+      "isSystem": false,
+      "isDefault": false,
+      "deleted": false,
+      "name": "Owner",
+      "alias": "owner",
+      "id": "9f4d940d-708d-4784-98bd-926195691e34",
+      "isFixed": true,
+      "defaultScope": "ENV"
+    },
+    "membership": "MANY_USERS",
+    "deleted": false,
+    "organization": {
+      "name": "organization-name",
+      "id": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e",
+      "entryPoint": "system"
+    },
+    "name": "environment-name",
+    "id": "a83e131f-4114-432b-a3fb-fd4c58907860",
+    "allowExternalMembers": false,
+    "state": "PENDING",
+    "serviceConnection": {
+      "id": "95ed7fa3-6f51-405e-b2c3-de43f20de2c3"
+    }
```

Comment:
I suggest that we reorganize the fields to make it more readable. Also updated the default role.

```suggestion
    "id": "a83e131f-4114-432b-a3fb-fd4c58907860",
    "name": "environment-name",
    "allowExternalMembers": false,
    "state": "PENDING",
    "membership": "MANY_USERS",
    "serviceConnection": {
      "id": "95ed7fa3-6f51-405e-b2c3-de43f20de2c3"
    },
    "defaultRole": {
      "id": "b7c48102-350e-40b9-9a3e-b7a0b3cfa535",
      "name": "Owner",
      "isSystem": false,
      "isDefault": false,
      "isFixed": true,
      "defaultScope": "ENV",
      "creationDate": "2020-01-02T16:04:27.000Z"
    },
    "organization": {
      "name": "organization-name",
      "id": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e",
      "entryPoint": "system"
    }
```